src/routes.tsx

```diff
@@ -0,0 +1,60 @@
+import ServiceGraphRouteHandler from './pages/ServiceGraph/ServiceGraphRouteHandler';
+import ServiceListPage from './pages/ServiceList/ServiceListPage';
+import IstioConfigPage from './pages/IstioConfigList/IstioConfigListPage';
+import ServiceJaegerPage from './pages/ServiceJaeger/ServiceJaegerPage';
+import ServiceDetailsPage from './pages/ServiceDetails/ServiceDetailsPage';
+import IstioConfigDetailsPage from './pages/IstioConfigDetails/IstioConfigDetailsPage';
+
+const baseName = '/';
+
+/**
+ * Return array of objects that describe vertical menu
+ * @return {array}
+ */
+const routes = () => [
+  {
+    iconClass: 'fa pficon-topology',
+    title: 'Graph',
+    to: '/service-graph/all',
+    redirect: true,
```

Comment:
Can we have a test that checks that we only have one item with `redirect` ? If we accidentally add more it will produce something undesired. Other option is to have a separate item in `routes.ts` e.g. `const defaultRoute = routes[0]` or more explicity:
```typescript
const serviceGraphRoute = {
  iconClass: 'fa pficon-topology',
  title: 'Graph',
  to: '/service-graph/all',
  redirect: true,
  component: ServiceGraphRouteHandler,
  pathsActive: [/\/service-graph\/(.*)\//g]
};
const defaultRoute = serviceGraphRoute;

const routes = () => [
  serviceGraphRoute,
  // etc ...
];
```

Reply:
Changed